src/Components/App/App.js

```diff
@@ -14,34 +14,56 @@ const App = () => {
   const [favorites, setFavorites] = useState([])
   const [loading, setLoading] = useState(true)
   const [error, setError] = useState(null)
-
-const thisData = mockData.data.map((data) => {
-  
-})
+  const [truckLocation, setTruckLocation] = useState([])
+  const [userLocation, setUserLocation] = useState('')
```

Comment:
Instead of userLocation being empty by default, I think we can hard-code a single user's lat/long data here.